src/ResourceTemplateValidator.js

```diff
@@ -143,11 +143,13 @@ const validateUniqueResourceURIs = async (resourceTemplate) => {
     const refs = propertyTemplate.valueConstraint?.valueTemplateRefs || []
     const resourceURIs = {}
     await Promise.all(refs.map(async (resourceTemplateIdRef) => {
-      const resourceTemplateRef = await fetchResourceTemplate(resourceTemplateIdRef)
-      if (resourceTemplateRef) {
-        const resourceURI = resourceTemplateRef.resourceURI
-        if (!resourceURIs[resourceURI]) resourceURIs[resourceURI] = []
-        resourceURIs[resourceURI].push(resourceTemplateIdRef)
+      if (resourceTemplateIdRef) { // ignore blank referenced templates elements
```

Comment:
For clarity, please consider adding this as a `filter()` on the refs. For example: `refs.filter(resourceTemplateIdRef => !_.isEmpty(resourceTemplateIdRef).map(...)`.